conntrack_linux.go

```diff
@@ -474,10 +491,22 @@ func (f *ConntrackFilter) AddProtocol(proto uint8) error {
 	return nil
 }
 
+// AddLabel adds the label value to the conntrack filter
+func (f *ConntrackFilter) AddLabel(tp ConntrackFilterType, labels [][]byte) error {
```

Comment:
@tssurya The comment seems a bit misleading; it actually adds either zero or more label values right? Should it be `AddLabels` instead?

Reply:
oh good point! let me fix this; yes it should be `AddLabels`